be/src/io/cache/dummy_file_cache.h

```diff
@@ -36,7 +36,7 @@ class DummyFileCache final : public FileCache {
 
     Status close() override { return Status::OK(); }
 
-    Status read_at(size_t offset, Slice result, size_t* bytes_read) override {
+    Status read_at(size_t offset, Slice result, size_t* bytes_read, const IOContext* ctx) override {
```

Comment:
use reference not pointer. 
Pointer is used as output parameter. 
and please add ctx before bytes_read.